packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -56,15 +57,38 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
       }),
       ['radioInnerCircleSize', 'radioVisibility', 'radioFill'],
     ),
+    content: buildProps(
+      () => ({
+        style: {
+          alignItems: 'flex-start',
+          flexDirection: 'column',
+        },
+      }),
+      [],
+    ),
     label: buildProps(
-      (tokens: RadioTokens) => ({
+      (tokens: RadioTokens, theme: Theme) => ({
         variant: tokens.variant,
         style: {
           marginTop: 2,
+          marginRight: tokens.marginRight,
+          color: tokens.color,
+          ...fontStyles.from(tokens, theme),
+        },
+      }),
+      ['color', 'marginRight', ...fontStyles.keys],
+    ),
+    subtext: buildProps(
+      (tokens: RadioTokens) => ({
+        variant: tokens.subtextVariant,
+        style: {
+          marginTop: tokens.marginTop,
+          marginRight: tokens.marginRight,
+          marginBottom: tokens.marginBottom,
           color: tokens.color,
         },
       }),
-      ['color'],
+      ['variant', 'marginTop', 'marginRight', 'marginBottom', 'color', 'subtextVariant'],
```

Comment:
variant doesn't need to be part of the cache key since it's not being used here.

Reply:
Removed!